src/compas/geometry/_primitives/line.py

```diff
@@ -337,7 +338,32 @@ def transformed(self, T):
         line.transform(T)
         return line
 
+    def divide_by_count(self, number=10, include_ends=False):
+        """Return list of points from dividing the line by specific number of divisions
+
+        Parameters
+        ----------
+        number : integer
+            number of divisions
+        includeEnds : boolean
+            True if including start and end point in division points
+            False if not including start and end point in division points
+
+        Returns
+        -------
+        list of: compas.geometry.Point // Point as sequence of values xyz)
```

Comment:
The formatting is incorrect, which I corrected below, but even then, the text is not super clear to me, "Point as sequence"? or "Sequence of points"?

```suggestion
        list of :class:`compas.geometry.Point`
            Point as sequence of values xyz
```